platform/spec/services/temporal/activities/register_workflow_run_activity_spec.rb

```
@@ -0,0 +1,108 @@
+      it 'raises error' do
+        expect {
+          subject.execute(
+            sync_run_id: sync_run.id,
+            workflow_id: workflow_id,
+            run_id: run_id
+          )
+        }.to raise_error("Workflow ID #{workflow_id} already exists")
+      end
+    end
+
+    context 'with multiple workflow registrations' do
```

rubocop (1.68.0)

[convention] Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.

(Style/StringLiterals)